web/channels/retro_channel.ex

```diff
@@ -60,6 +65,22 @@ defmodule RemoteRetro.RetroChannel do
     {:noreply, socket}
   end
 
+  def handle_in("new_action_item", %{"body" => body, "assigneeId" => assignee_id, "userId" => user_id}, socket) do
```

Comment:
Looks like we're repeating ourselves here, too, as the handler for `"new_idea"` does almost the exact same thing as this new handler, except it expects an `assigneeId` and hard-code's the category as `"action-item"`